webapp/package.json

```diff
@@ -6,7 +6,8 @@
   "scripts": {
     "build": "webpack --mode=production",
     "lint": "eslint --ignore-pattern node_modules --ignore-pattern dist --ext .js --ext .jsx . --quiet",
-    "fix": "eslint --ignore-pattern node_modules --ignore-pattern dist --ext .js --ext .jsx . --quiet --fix"
+    "fix": "eslint --ignore-pattern node_modules --ignore-pattern dist --ext .js --ext .jsx . --quiet --fix",
+    "test": "echo \"no test specified\""
```

Comment:
Should we complete this test script definition with the current setup with have with jest in the webapp? I'm wondering if it will be better to leave that as a boilerplate over this message  🤔

Reply:
Agree with @marianunez -- I think we need this to be more fleshout, otherwise we're just kicking the problem down into the stack.

@kaakaa, would you be open to:
1) Updating the package dependencies to rely on jest, and changing this target to run jest appropriately?
2) Add a very simple test that could be used as a starting point for plugin authors?